Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```diff
@@ -42,25 +42,25 @@
     contents:
       - id: FlashlightSeclite
       - id: WeaponDisabler
-        prob: 0.3
+#        prob: 0.3 # DeltaV - just give them the damn disabler
       - id: ClothingBeltSecurityFilled
       - id: Flash
       - id: ClothingEyesGlassesSecurity
       - id: ClothingHeadsetAltSecurity
       - id: ClothingHandsGlovesCombat
       - id: ClothingShoesBootsJack
-      - id: ClothingOuterCoatWarden
```

Comment:
GOD PLEASE replace them with something in a similar style, do not just remove them